umbraco-cloud/set-up/project-settings/secrets-management.md

```diff
@@ -138,6 +138,7 @@ While we have a number of prefixes that are not accepted, we do allow the use of
 * `Umbraco__Forms__FieldTypes__Recaptcha`
 * `Umbraco__CMS__Integrations`
 * `Umbraco__CMS__DeliveryAPI`
+* `UMBRACO__LICENSES__YOURPRODUCT`
```

Comment:
```suggestion
* `UMBRACO__LICENSES__`
```

Reply:
@pijemcolu wouldnt it be more clear to customers that they can add it for the different products with YOURPRODUCT at the end?

They might want to have more than one license in their projects.

Reply:
We could do `UMBRACO__LICENSES__XX` as an alternative.
And then in a note below, we mention how to replace XX with the product name and give an example or two?

Reply:
It's a set of prefixes. This is a set of prefixes:
```
* `Umbraco__CMS__Global__Smtp__`
* `Umbraco__Forms__Security__FormsApiKey`
* `Umbraco__Forms__FieldTypes__Recaptcha`
* `Umbraco__CMS__Integrations`
* `Umbraco__CMS__DeliveryAPI`
* `UMBRACO__LICENSES`
```

Either they're all prefixes, or we shall adjust all of the above to be something such as: 
```
* `Umbraco__CMS__Global__Smtp__*`
* `Umbraco__Forms__Security__FormsApiKey__*`
* `Umbraco__Forms__FieldTypes__Recaptcha__*`
* `Umbraco__CMS__Integrations__*`
* `Umbraco__CMS__DeliveryAPI__*`
* `UMBRACO__LICENSES__*`
```

Reply:
Ah cool! 💪 
So _anything_ can be added after the prefix? Is that correctly understood?
I could do something like `UMBRACO__LICENSES__MyCoolUmbracoLicense` ? 😅

Reply:
Yes, that's the way this is coded within the Umbraco Cloud platform, you can add any string behind these prefixes.

```
* `Umbraco__CMS__Global__Smtp__`
* `Umbraco__Forms__Security__FormsApiKey`
* `Umbraco__Forms__FieldTypes__Recaptcha`
* `Umbraco__CMS__Integrations`
* `Umbraco__CMS__DeliveryAPI`
* `UMBRACO__LICENSES`
```

Reply:
Excellent. Let's keep that format then 💪 
@jonat123 - you up for making the changes to the PR?